### ISSUE TITLE
Deselecting Cover Page causes errors

### ISSUE THREAD

**Issue by Ambrosiani** [tainacan/tainacan#431]
**Bug description**

When I select a custom Cover Page for a collection, that url is replaced by a collection starting page. However, when I deselect the page, the old wordpress page does not reappear.

**How to reproduce such bug**

Use the following template to create a step-by-step guide:
1. Go to wp-admin/admin.php?page=tainacan_admin#/collections/[ID]/settings
2. Turn on "Cover Page"
3. Save
4. Edit collection settings again – deselect Cover Page OR remove the selected page OR delete the collection

**Expected behavior**

The existing wordpress page should be visible again

**Screenshots**

**Comment by Ambrosiani** [comment]
by manually changing the slug of the wordpress page, it becomes visible again. So I'm guessing the slug is saved somewhere, and is not removed correctly.

**Comment by mateuswetah** [comment]
Uhmm, at first I thought it was something related to an old issue (#351) that we've closed recently... But that seems different. @vnmedeiros can you check the slug suggestion?

**Comment by mateuswetah** [comment]
Hi @Ambrosiani, we've facing difficulties on reproducing this error, at least here all seems fine...

Just to clarify a few things, maybe the behavior is not clear:

- There is the **Collection Items list** page, which you can always find in `/slug-of-collection`;
- If you don't change anything, every link to `/collections/slug-of-collection` will redirect to `/slug-of-collection`, aka the Collection Items list. You can fin this link in the admin page (the eye icons) or in the theme side (`/collections`, the **Collections list**);
- When you configure a **Collection cover page**, the `/collections/slug-of-collection` address will render the same content of the selected page. Notice that the Collection Items list (`/slug-of-collection`) and the selected page continue to exist individually, but you can now access the page via Collections list for example (`/collections`). 

One thing that often causes confusion is if you create _a page and users the same name of the collection_. In that case, the address will conflict with the Collection Items list page name (`/slug-of-collection`)! The slug setting in the Collection form is there exactly to avoid this. It should be able to detect slugs that are already being used. It seems that the WordPress page editor, although, can't detect the other way around.

Am I right @leogermani?

**Comment by Ambrosiani** [comment]
@mateuswetah @leogermani hi,

I realized my mistake now. I used the same slug both for a wordpress page **and** for a tainacan collection. The tainacan collection slug then overrode the wordpress page slug, causing the page to disappear. So this was related to the slug settings, not to the cover page settings. I was experimenting with the cover page at the same time which is why I thought that was the origin of the error.
